### PR TITLE
dynamic sca.py

### PR DESCRIPTION
**Pull request by hashanant**

### CODE REVIEW

File: stdplugins/sca.py [hunk 1]
```diff
@@ -1,19 +1,29 @@
...
-Options: typing, contact, game, location, voice, round, video, photo, document, cancel"""
+Syntax: .sca <option> <time in sec>
+        .scaoptions"""
 
 import asyncio
 from uniborg.util import admin_cmd
 
 
-@borg.on(admin_cmd(pattern="sca ?(.*)"))
+@borg.on(admin_cmd("sca (.*) (.*)"))
```
Comment (SpEcHiDe) [inline stdplugins/sca.py]: you need a pattern= here and in all admin_cmd decorators

Comment (hashanant) [reply stdplugins/sca.py]: oops I forgot that

**Comment by SpEcHiDe** [comment]
why you closed PR?

**Comment by hashanant** [comment]
I thought it's done..!
I committed those changes

**Pull request closed without merging**